src/amd_functions.jl

```diff
@@ -1,9 +1,11 @@
-global const _libamd = Libdl.dlopen("libamd")
+function __init__()
```

Comment:
Interesting! How on Earth did you figure that out?

Reply:
`_libamd` was null from the start, so the problem was in this line. Then I remembered `__init__`, which is kind of a Module constructor, which runs at runtime. However, I can't explain fully why this works, and why it wasn't needed before. I'm guessing it has to do with the compilation of Libdl which is a separate package in 0.7.

Reply:
fantastic! Thank you!